src/sgd/frontend/yeastgenome/static/templates/snapshot.jinja2

```diff
@@ -50,7 +50,7 @@
 				</h2>
 				<hr />
 				<p>
-					These graphs represent the state of Gene Ontology (GO) annotation of the entire genome using GO slims, which are high-level subsets of GO terms that allow grouping of genes into broad categories. Each graph represents a single aspect of GO, such as Biological Process (BP), Molecular Function (MF), or Cellular Component (CC). The radio buttons can be used to access the bar graphs for MF and CC.  The GO slim terms are listed, along with the number of gene products annotated to that term or to a child of that term. Only annotations made by manually curated and high-throughput methods are included. Note that some gene products are annotated to one or more GO terms that map to more than one GO slim term. The GO data summarized in these graphs below are available in the go_slim_mapping.tab file on <a href="http://www.yeastgenome.org/downloads/curation">SGD's Downloads Site</a>.
+					These graphs represent the state of Gene Ontology (GO) annotation of the entire genome using GO slims, which are high-level subsets of GO terms that allow grouping of genes into broad categories. Each graph represents a single aspect of GO, such as Biological Process (BP), Molecular Function (MF), or Cellular Component (CC). The radio buttons can be used to access the bar graphs for MF and CC.  The GO slim terms are listed, along with the number of gene products annotated to that term or to a child of that term. Only annotations made by manually curated and high-throughput methods are included. Note that some gene products are annotated to one or more GO terms that map to more than one GO slim term. The GO data summarized in these graphs below are available in the go_slim_mapping.tab file on <a href="https://downloads.yeastgenome.org/curation/">SGD's Downloads Site</a>.
```

Comment:
and let's change this to /search?category=download&page=0&q=go_slim_mapping&status=Archived